setup.py

```diff
@@ -44,7 +44,6 @@ def main():
             "loopy>=2019.1",
             "dataclasses; python_version<'3.7'",
             "types-dataclasses",
-            "pytato>=2021.1",
```

Comment:
Thanks! I agree... no need for a hard dependency. `arraycontext` works just fine without pytato.